tldw_Server_API/tests/Audio/test_ws_fallbacks.py

```
@@ -0,0 +1,104 @@
+
+
+@pytest.mark.asyncio
+async def test_model_unavailable_triggers_fallback_warning(monkeypatch):
+    # Force Parakeet core variant builder to return None so adapter initialize fails
+    import tldw_Server_API.app.core.Ingestion_Media_Processing.Audio.Parakeet_Core_Streaming.transcriber as core_tx
+    monkeypatch.setattr(core_tx, "_variant_decode_fn", lambda m, v: None)
+
+    # Enable fallback to Whisper
+    import tldw_Server_API.app.core.Ingestion_Media_Processing.Audio.Audio_Streaming_Unified as unified
+    monkeypatch.setattr(unified, "load_comprehensive_config", lambda: _make_cfg(True))
+    monkeypatch.setattr(unified, "get_whisper_model", lambda size, device: _FakeWhisperModel())
```

Ruff (0.14.2)

Unused lambda argument: `size`

(ARG005)